readthedocs-theme/templates/readthedocs/includes/announcements.html

```diff
@@ -1,51 +1,53 @@
-              <div><span class="ui label primary">News</span></div>
-              <h4>Our Sphinx theme version 1.0.0 is out</h4>
-              <p>
-                We have released a major version 1.0.0 of our Sphinx theme. This adds support for recent versions of Sphinx and docutils among other very cool things. Check out the blog post.
-              </p>
-              <p>
-                <a href="https://blog.readthedocs.com/newsletter-september-2021/">Read more</a>
-              </p>
+<section class="ui vertical segment" id="announcements">
+  <div class="ui container">
+    <div class="ui raised segment">
+      <div class="ui grid stackable three column">
+        
+        {% for article in articles if 'homepage' in article.tags %}
+          {% if loop.index <= 3 %}
+            <div class="column">
+              <div class="ui basic segment">
+                <div>
+                  <span class="ui label {% if article.category == 'Announcement' %}red{% else %}primary{% endif %}">
+                    {# article.category defaults to 'posts' when no category is provided #}
+                    {% if article.category == 'posts' %}
+                      Article
+                    {% else %}
+                      {{ article.category }}
+                    {% endif %}
+                  </span>
+                </div>
+                <h4>{{ article.title }}</h4>
+                <p>
+                  {% if article.description %}
+                    {{ article.description|striptags|escape }}
+                  {% else %}
+                    {{ article.summary|striptags|escape }}
+                  {% endif %}
+                </p>
+                <p>
+                  <a href="/{{ article.url }}">Read more</a>
+                </p>
+              </div>
             </div>
-          </div>
-          <div class="column">
-            <div class="ui basic segment">
-              <div><span class="ui label">Update</span></div>
-              <h4>You are all up to date</h4>
-              <p>
-                We post important announcements, news and updates in this area. You can also check out our <a href="https://blog.readthedocs.com/">blog</a> or subscribe the newsletter to receive future updates first hand, plus receive our monthly report.
-              </p>
-              <p>
-                <a href="#">Subscribe our newsletter</a>
-              </p>
+          {% endif %}
+          {% if loop.last and loop.length < 3 %}
+            <div class="column">
+              <div class="ui basic segment">
+                <div><span class="ui label">Update</span></div>
+                <h4>You are all up to date</h4>
+                <p>
+                  We post important announcements, news and updates in this area. You can also check out our <a href="https://blog.readthedocs.com/">blog</a> or subscribe the newsletter to receive future updates first hand, plus receive our monthly report.
```

Comment:
The blog link should be to the internal blog.

```suggestion
                  We post important announcements, news and updates in this area. You can also check out our <a href="/blog/">blog</a> or subscribe to our newsletter to receive future updates.
```